js/interface.js

```diff
@@ -141,7 +139,7 @@ dsQueryProvider.then(function onForwardDsQueryProvider(result) {
       validation.email = {
         toColumn: result.data.columns.emailMatch,
         matchColumn: result.data.columns.emailMatch,
-        template: emailProviderResult || defaultEmailSettings,
+        template: validation.email ? validation.email.template : defaultEmailSettings,
```

Comment:
is `emailProviderResult` not used anymore after this patch?

Reply:
@squallstar Yes, we are saving `emailProviderResult` to the `dataSource` directly.